src/Resources/Resources/ChangeLog.md

```diff
@@ -21,6 +21,7 @@
 ## Upcoming Release
 * Used utf8 encoding for reading stdout & stderr when invoking Bicep. [#23246]
 * Fixed regression in `Publish-AzBicepModule` [Azure/bicep/12461](https://github.com/Azure/bicep/issues/12461)
+* Added `-WithSource` parameter to `Publish-AzBicepModule` for publishing source with a module (currently experimental)
```

Comment:
Please insert change log between L21-L22, you may find conflicts then. Please resolve them.